cmd/yaml.go

```diff
@@ -33,6 +38,46 @@ func ParseYaml(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interrup
 	return cfg, nil
 }
 
+// LookupManifestFile look up for a given file
+// in the current working directory
+func LookupManifestFile(fname string, p *cookoo.Params) (interface{}, cookoo.Interrupt) {
```

Comment:
So the way I read this, if I do `-y foo.yml` and I have two files, one named `foo.yml` and one named `foo.yaml`, then `foo.yaml` will get loaded even though I explicitly said `foo.yml`, right?